packages/agent-routing/hooks/route_config.py

```
@@ -26,7 +26,7 @@
     load_cli_tools_config,
 )
 
-# エージェントルーティング設定（25エージェント分）
+# エージェントルーティング設定（28エージェント分）
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)